services/frontend-v3/src/components/education/EducationView.jsx

```diff
@@ -17,6 +17,19 @@ const EducationView = ({ educationInfo }) => {
     },
   };
 
+  const getUrl = (item) => {
+    if (item.attachmentLinks && item.attachmentLinks.length > 0)
+      return item.attachmentLinks.map((i) => (
+        <Tag color="rgb(114, 114, 114)" key={i.id}>
+          <LinkOutlined />
+          <a target="_blank" rel="noreferrer" href={i.url}>
+            {i.name}
+          </a>
+        </Tag>
```

Comment:
Here I would make the tag clickable and I figured out to add proper spacing between the icon and the text, you need to wrap the text with a `<span></span>`

Maybe something like this?
```suggestion
        <a target="_blank" rel="noreferrer" href={i.url}>
          <Tag color="rgb(114, 114, 114)" key={i.id}>
            <LinkOutlined />
            <span>
              {i.name}
            </span>
          </Tag>
        </a>
```